src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/Strings/en-us/Resources.resw

```diff
@@ -0,0 +1,140 @@
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="InfoBarCloseButtonName" xml:space="preserve">
+    <value>Close</value>
+    <comment>Automation name of the close button.</comment>
+  </data>
+  <data name="InfoBarCloseButtonTooltip" xml:space="preserve">
+    <value>Close</value>
+    <comment>The tooltip that appears on the Close button in InfoBar.</comment>
+  </data>
+  <data name="InfoBarClosedNotification" xml:space="preserve">
+    <value>InfoBar dismissed</value>
+    <comment>The string read by narrator when the InfoBar closes.</comment>
+  </data>
+  <data name="InfoBarCustomLandmarkName" xml:space="preserve">
+    <value>InfoBar</value>
+    <comment>This is the custom landmark used to denote an InfoBar to narrator.</comment>
+  </data>
+  <data name="InfoBarOpenedNotification" xml:space="preserve">
+    <value>%1!s! %2!s!</value>
```

Comment:
Again we have format strings that won't work in c# I think. We need a plan for this. ColorPicker has the same unresolved problem. We probably need a special string.FormatWinUI() method that will convert from cpp-style to c#-style and then pass to string.Fromat().

Reply:
Agreed, this needs to be adjusted. I didn't notice that :-D .

Reply:
@robloo turns out this is in fact already handled in Uno :-D  - `Uno.UI.Helpers.WinUI.StringUtil.cs`

Reply:
Great, I dont think many knew about that! I'll take a look then update ColorPicker as well.